env.arolla.sh

```diff
@@ -305,6 +310,7 @@ EOF
         cat > $ENVIRONMENT_TEMPFILE <<-EOF
             # Generated with the build script
             # implicit module purge
+            module load cmake/3.14.3
```

Comment:
this is in the COSMO_TESTENV section - for tsa with RH7.6 - module should be 
cmake/3.14.5

Reply:
I don't see any reference to `tsa` or a `env.tsa.sh` ...

Reply:
you need to do
       module unuse /apps/arolla/UES/modulefiles
        module use /apps/arolla/UES/jenkins/RH7.6/generic/easybuild/modules/all
to see the correct modules on tsa

Reply:
Ok but there is still no `env.tsa.sh` ... so is there a dirty trick somewhere and it loads the same env file as arolla?